README.md

```diff
@@ -213,7 +213,9 @@ Examples:
   etrace --kernel Process --where ProcessName=myProcessName
   etrace --kernel Process --event Thread/Stop --where ProcessId=4
   etrace --kernel Process --event Thread/Stop --where ThreadId=10272	
+  etrace --kernel Process --event --where "ThreadId > 1 && ... && ProcessName=git-lfs"
```

Comment:
changed to :   etrace --kernel Process --event --where "ThreadId > 1 && ProcessName=git-lfs"